pkg/controller/hawtio/webconsole.go

```diff
@@ -0,0 +1,50 @@
+package hawtio
```

Comment:
I'd suggest to move that logic into the `openshift` package, as this is where the other methods related to OpenShift CR creation are.

Reply:
thank you for the suggestion.
I am working on to finalize another func in operator-utils, I will get back to you on this asap.

Reply:
@astefanutti   
ready for review.